index.js

```diff
@@ -62,17 +62,19 @@ function inject (dispatchFunc, options, callback) {
     }
   }
 
+  const that = options.server || {}
```

Comment:
can you call it `server`? `that` seems `this`.

Reply:
done